.github/workflows/linux-check.yaml

```diff
@@ -21,6 +21,7 @@ on:
       - '!tools/python/testserver.py'
       - '!tools/python/SiblingKiller.py'
       - xcode/**
+      - qt/res/app.organicmaps.desktop.metainfo.xml
```

Comment:
Is qt/res/ a proper location for this metadata file? What if other metadata files will be added later?

Reply:
> Is qt/res/ a proper location for this metadata file?

It is packaging related, so perhaps it could go under `packaging/`?

Reply:
Yes!